raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
It is raising an exception. You are correct that `assert not` is more encompassing than the much more specific `is not None`. 

BUT the function `manager_address_if_token_registered()` returns either an address or None and not `0` or `''`. So in this case we are perfectly fine.

The exception thrown is a `TransactionFailed` since it's asking the blockchain if there is a channel manager for this Token. That is the exception I am catching.

I will make it more explicit so that all people who read the code can figure it out faster. Thank you for your comments.